src/functionaltest/resources/functional-test.properties

```diff
@@ -0,0 +1,2 @@
+test.selenium.firefox.BZip2File.linux = https://download.mozilla.org/?product=firefox-61.0.1-ssl&os=linux64&lang=en-GB
+test.selenium.firefox.BZip2File.linux.name = firefox-61.0.1.tar.bz2
```

Comment:
This property can be removed and extract filename from the url.

Reply:
Done